### PR TITLE
Preprocess

### PR DESCRIPTION
**Pull request by Swendart**
Add a preprocess method to cut the dataset in a train and test dataset with the following requirements 
- Length of train dataset and test_dataset is a fix amount (like 100 000 lines)
- Respect ratio between train and test
- Have an equirepartition of lines for each anomalies 
- Shuffle data

### CODE REVIEW

File: src/parser/parser.py [hunk 1]
```diff
@@ -43,8 +44,19 @@ def __init__(self,path_save="../data/output/temp/",log_path="../data/output/pars
...
+        preprocess = Preprocess(file_name)
+        preprocess.create_label_content()
+
+        train_dataset_path = self.path_save + "train_dataset.csv"
+        test_dataset_path = self.path_save + "test_dataset.csv"  
+
+        preprocess.create_train_test_dataset(train_dataset_path, test_dataset_path, clustering_length, ratio)
+
+        # Load the train and test datasets
+        train_data = self.parse_kdd(train_dataset_path)
```
Comment (ManuSerp) [inline src/parser/parser.py:57]: J'ai changé la

**Review by ManuSerp — APPROVED** [review]
